### ISSUE TITLE
New authentification supports

### ISSUE THREAD

**Issue by Neustradamus** [bytebuddha/libsip#4]
Can you add new authentification supports?
- https://tools.ietf.org/html/draft-ietf-sipcore-digest-scheme-15

Note: MD5 is obsolete since a long time

**Comment by bytebuddha** [comment]
Yes i plan on it soon. Im going to move most of the authentication login into the AuthHeader Struct. That is what i was planning on doing after completing the Invite helpers

**Comment by bytebuddha** [comment]
I implemented the sha256 and sha512 algorithym's. It's still a very basic implementation but usable.

**Comment by Neustradamus** [comment]
Thanks a lot! :)

Do not hesitate to improve the "very basic implementation" ^^
